src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,12 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        Optional<User> optionalUser = Optional.ofNullable(user);
+
+        return optionalUser
+                .map(User::getAge)
+                .filter(age -> age >= 30 && age <= 45)
```

Comment:
```suggestion
                .filter(User::허용검증)
```

검증에대한 의미있는 이름을 지어보고,
객체에게 물어볼수도 있겠네요! 😄